### PR TITLE
On location changed callback

### PR DESCRIPTION
**Pull request by dydkask**

### CODE REVIEW

File: packages/atlas/lib/src/atlas.dart [hunk 1]
```diff
@@ -33,6 +33,9 @@ class Atlas extends StatelessWidget {
   /// which can be used to manipulate the map.
   final ArgumentCallback<AtlasController> onMapCreated;
 
+  /// 'onLocationChanged' Callback that receives a position when the user moves
```
Comment (felangel) [inline packages/atlas/lib/src/atlas.dart]:
```suggestion
  /// `onLocationChanged` callback which receives a position when the user moves
```

File: packages/atlas/lib/src/atlas.dart [hunk 2]
```diff
@@ -33,6 +33,9 @@ class Atlas extends StatelessWidget {
...
+  final ArgumentCallback<LatLng> onLocationChanged;
```
Comment (felangel) [inline packages/atlas/lib/src/atlas.dart:41]: So the map component is responsible for tracking the user's location in real-time?

Comment (dakrawczyk) [reply packages/atlas/lib/src/atlas.dart:41]: @felangel yes. For example the HERE provider uses a location engine supplied by the HERE SDK. This engine is responsible for receiving updates from the user's location if permission has been given to the client application. If the client application has `showMyLocation` set to true, the mapProvider will intercept this location update and use it to show the user's current location on the map.

**Review by dakrawczyk — DISMISSED** [review]

**Review by tchabot22 — DISMISSED** [review]

**Review by tchabot22 — DISMISSED** [review]

**Comment by dakrawczyk** [comment]
Fixing conflict..

**Review by tchabot22 — APPROVED** [review]

**Review by dakrawczyk — APPROVED** [review]

**Pull request merged**